codecov.yml

```diff
@@ -40,7 +40,8 @@ coverage:
       default:
         branches: null
 
-  ignore: null
+  ignore:
+    - tests/*
```

Comment:
This configuration change is outside the scope of this bugfix. Can I suggest, we add those changes in a separate PR?

Reply:
Fair enough. I created #377 and will now remove it from this PR.

Reply:
Thanks!